### PR TITLE
Auto Test Script for many new CI

### PR DESCRIPTION
**Pull request by d-jana**
##### SUMMARY
Auto Test Script for many new CI for labs, rhpds and RHTR

##### ISSUE TYPE
- Feature Pull Request

### CODE REVIEW

File: tests/jenkins/collections-lab.groovy [hunk 1]
```diff
@@ -0,0 +1,300 @@
...
+                            -d '{\"username\": \"jenkins\", \"icon_url\": \"https://dev-sfo01.opentlc.com/static/81c91982/images/headshot.png\", \"text\": \"@here :rage: ${env.JOB_NAME} (${env.BUILD_NUMBER}) failed retiring ${guid}.\"}'\
+                            """.trim()
+                        )
+                    }
+                }
+            }
+        }
+        stage('Wait for deletion email') {
+            steps {
+                git url: 'https://github.com/sborenst/ansible_agnostic_deployer',
```
Comment (fridim) [inline tests/jenkins/collections-lab.groovy:223]: You should use the new URLs

not https://github.com/sborenst/ansible_agnostic_deployer  but https://github.com/redhat-cop/agnosticd

File: tests/jenkins/collections-lab.groovy [hunk 2]
```diff
@@ -0,0 +1,300 @@
...
+def opentlc_creds = 'b93d2da4-c2b7-45b5-bf3b-ee2c08c6368e'
+def opentlc_admin_creds = '73b84287-8feb-478a-b1f2-345fd0a1af47'
+def cf_uri = 'https://portal.opentlc.com'
+def cf_group = 'opentlc-access-cicd'
+// IMAP
+def imap_creds = 'd8762f05-ca66-4364-adf2-bc3ce1dca16c'
+def imap_server = 'imap.gmail.com'
+// Notifications
+def notification_email = 'gpteinfrasev3@redhat.com'
+def rocketchat_hook = '5d28935e-f7ca-4b11-8b8e-d7a7161a013a'
```
Comment (fridim) [inline tests/jenkins/collections-lab.groovy:12]: You can get rid of everything related to Rocketchat

File: tests/jenkins/acm-cluster-hub.groovy [hunk 3]
```diff
@@ -0,0 +1,304 @@
...
+                    }
+                    withCredentials([string(credentialsId: rocketchat_hook, variable: 'HOOK_URL')]) {
+                        sh(
+                            """
+                            curl -H 'Content-Type: application/json' \
+                            -X POST '${HOOK_URL}' \
+                            -d '{\"username\": \"jenkins\", \"icon_url\": \"https://dev-sfo01.opentlc.com/static/81c91982/images/headshot.png\", \"text\": \"@here :rage: ${env.JOB_NAME} (${env.BUILD_NUMBER}) failed retiring ${guid}.\"}'\
+                            """.trim()
+                        )
+                    }
```
Comment (fridim) [inline tests/jenkins/acm-cluster-hub.groovy:221]: rocketchat stuff can be removed in all pipelines

Comment (d-jana) [reply tests/jenkins/acm-cluster-hub.groovy:221]: Removed from this file. Will remove from all file next week.

File: tests/jenkins/collections-lab.groovy (hunk 2 above)
Comment (d-jana) [reply tests/jenkins/collections-lab.groovy:12]: Removed from this file. Will remove from all file next week.

File: tests/jenkins/collections-lab.groovy (hunk 1 above)
Comment (d-jana) [reply tests/jenkins/collections-lab.groovy:223]: Updated on this file. Will remove from all file next week.

**Review by bosebc — APPROVED** [review]
Looks good. Thanks

**Pull request merged**